authenticate/handlers.go

```diff
@@ -151,12 +140,12 @@ func (a *Authenticate) VerifySession(next http.Handler) http.Handler {
 			return a.reauthenticateOrFail(w, r, err)
 		}
 
-		if state.dataBrokerClient != nil {
```

Comment:
I might be missing something here but it feels more clear to err on a nil databroker client, not continue?